persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/conditions/ConditionESQueryBuilderDispatcher.java

```diff
@@ -83,12 +83,12 @@ public QueryBuilder buildFilter(Condition condition, Map<String, Object> context
             if (contextualCondition != null) {
                 return queryBuilder.buildQuery(contextualCondition, context, this);
             }
-        }
-
-        // if no matching
-        logger.warn("No matching query builder. See debug log level for more information");
-        if (logger.isDebugEnabled()) {
-            logger.debug("No matching query builder for condition {} and context {}", condition, context);
+        } else {
```

Comment:
Don't we want to add a log when no matching condition happen too?

Reply:
You mean when we can't resolve the parameters ? No I don't think so that would happen too often.

Reply:
Indeed, I was talking about parameters, sounds like no matching signature in that case isn't it?

Reply:
It's just that the event doesn't have a value for a parameter, which can happen on a lot of events. I think it might be too much noise to add a log for that.

Reply:
ok, thanks for the answers !